src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    margin-left: 3px;
+    width: 160px;
+    height: 134px;
+    padding-bottom: 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: $main-font-weight;
+    line-height: 18px;
+    text-align: left;
+    margin: 0;
+  }
+
+  &__code {
+    padding-top: 4px;
+    font-size: 10px;
+    font-weight: $second-font-weight;
+    line-height: 14px;
+    color: $secondary-color;
+    margin: 0;
+  }
+
+  &__review {
+    display: flex;
+    padding-top: 17px;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+    margin-left: 2px;
+
+    &-text {
+      font-size: 10px;
+      font-weight: $second-font-weight;
+      line-height: 14px;
+      margin: 0;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 23px;
+    padding-bottom: 16px;
+    align-items: center;
+
+    &-title {
+      line-height: 18px;
+      font-weight: $second-font-weight;
+      color: $secondary-color;
+      margin: 0;
+    }
+
+    &-amount {
+      margin: 0;
+      font-size: 16px;
+      font-weight: $bold-font-weight;
+      line-height: 18px;
+    }
+  }
+
+  &__button-buy {
+    display: flex;
+    width: 166px;
+    height: 40px;
+    background-color: $blue-color;
+    border-radius: 5px;
+    border: none;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+
+    font-size: 14px;
+    font-family: $font;
+    font-weight: $bold-font-weight;
+    line-height: 16px;
+    color: $bg-color;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      border: 1px solid $blue-color;
+      animation: button 300ms linear forwards;
+    }
+  }
+
+  &:hover {
+    animation: scale 300ms linear forwards;
+  }
+
+  &:hover .card__name {
+    animation: title 300ms linear forwards;
```

Comment:
use transition for smoothness of pseudo-classes, and animation as independent animations